browser/ai_chat/android/ai_chat_utils.cc

```diff
@@ -0,0 +1,37 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/build/android/jni_headers/BraveLeoUtils_jni.h"
+
+#include "base/android/jni_android.h"
+#include "base/android/jni_string.h"
+#include "brave/components/ai_chat/core/common/buildflags/buildflags.h"
+#include "content/public/browser/web_contents.h"
+
+#if BUILDFLAG(ENABLE_AI_CHAT)
```

Comment:
minor notice, `#include "content/public/browser/web_contents.h"` can also be under this build flag